locales/de.json

```diff
@@ -166,6 +167,16 @@
 				"required": "erforderlich",
 				"signAgreement": "Du kannst auf dieser Seite im Namen deiner Schüler:innen das Einverständnis zur {{linkPrivacyPolicy}} und zur {{linkTermsOfUse}} der {{shortTitle}} erklären. <br>Durch den Klick auf <i>Einverständnis im Namen der Schüler:in erklären</i> bestätigst du, dass du das Einverständnis des Schülers/der Schülerin bzw. der Eltern auf anderem Wege erhalten hast.",
 				"theNextStepIsToInviteOrAdd": "Im nächsten Schritt kannst du Schüler:innen zur Klasse einladen oder hinzufügen."
+			},
+			"brb": {
+				"text": {
+					"noConsentNecessary": "Eine Einverständniserklärung bei der Registrierung von Schüler:innen muss <b>nicht</b> eingeholt werden. Die Nutzung der Schul-Cloud Brandenburg ist im brandenburgischen Schulgesetz (§ 65 Abs. 2 BbGSchulG) geregelt."
```

Comment:
I see why you hard coded the instance name, but for future usage the instance name should still use the variable